esmvaltool/cmorizers/obs/cmorize_obs_osi_409_sh.py

```diff
@@ -0,0 +1,25 @@
+# pylint: disable=invalid-name
+"""ESMValTool CMORizer for OSI-SAF data.
+
+Tier
+   Tier 2: other freely-available dataset.
+
+Source
+   http://osisaf.met.no/p/ice/
+
+Last access
+   20190502
+
+Download and processing instructions
+    Download the desired years from the following ftp:
+        ftp://osisaf.met.no/reprocessed/ice/conc/v1p2
```

Comment:
Is this correct? Why different links for `nh` and `sh`?

Reply:
As I understood, `v1p2` is `OSI-409` and `v2p0` is `OSI-450`.
And they contain both `nh` and `sh`.